redhat-ipaas-dev-single-tenant.yml

```diff
@@ -95,6 +95,11 @@ parameters:
   name: POSTGRESQL_VOLUME_CAPACITY
   required: true
   value: 1Gi
+  - description: Enables test-support endpoint on backend API
+    displayName: Test Support Enabled
+    name: TEST_SUPPORT_ENABLED
+    required: true
+    value: "true"
```

Comment:
ok, overlooked that it was only for the dev templates. makes probably sense then.